packages/components/src/Page/Page.tsx

```diff
@@ -51,6 +51,9 @@ export interface InterfacePage {
   )[];
   /** Labels for the title section */
   headerLabels?: (InterfaceLabel | "skeleton")[];
+
+  /** Provisional property to force the headerAction view in desktop mode */
+  headerActionsInDesktop?: boolean;
```

Comment:
Sugestão: para que os tipos do componente Responsive estejam ligados corretamente a essa propriedade é necessário fazer uma pequena alteração a tipagem dele, este componente se encontra no caminho `packages/components/src/Responsive/index.tsx:3` precisamos fazer a seguinte mudança 

```typescript
export type TypeDisplays = ("desktop" | "mobile" | "tablet")[];

export interface InterfaceResponsive {
  displays: TypeDisplays;
  children: React.ReactNode;
}
```


```suggestion
  headerDisplays?: TypeDisplays;
```